css/style.css

```diff
@@ -117,3 +117,28 @@ a {
     margin-right: 15px;
     padding-right: 15px;
   }
+
+  @media only screen and (max-width: 960px) {
```

Comment:
The responsive doesn't work, there are two ways to fix it.
1- replace `max-width` with `max-device-width`
or
2-  In the meta tag, add  `content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">`